tests/phpunit/data/requestCache/config.json

```diff
@@ -4,7 +4,7 @@
       "ttl": 7200
     },
     "api": {
-      "baseUrl": "http://time.jsontest.com/",
+      "baseUrl": "http://jsontest",
```

Comment:
Sluzba http://time.jsontest.com ma vypadok.

![image](https://user-images.githubusercontent.com/19371734/114138360-ceea0580-990d-11eb-96ba-e183e5213857.png)

Pouziva sa v 3 testoch, vracia to datum ako JSON response.
Premigroval som to na lokalnu Docker sluzbu, ktora robi to iste.